submissions/KittyHome/README.md

```diff
@@ -0,0 +1,104 @@
+# Kitty Home
```

Comment:
It'll be nice to have all the images fetched from somewhere instead of being included in the extension directly.